examples/webgpu_loader_materialx.html

```diff
@@ -100,19 +102,68 @@
 				const container = document.createElement( 'div' );
 				document.body.appendChild( container );
 
-				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 0.25, 50 );
-				camera.position.set( 0, 3, 20 );
-
-				scene = new THREE.Scene();
-
-				renderer = new THREE.WebGPURenderer( { antialias: true, alpha: true } );
+				renderer = new THREE.WebGPURenderer( { antialias: true } );
 				renderer.setPixelRatio( window.devicePixelRatio );
 				renderer.setSize( window.innerWidth, window.innerHeight );
 				renderer.toneMapping = THREE.LinearToneMapping;
 				renderer.toneMappingExposure = .5;
 				renderer.setAnimationLoop( render );
 				container.appendChild( renderer.domElement );
 
+				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 0.25, 200 );
+				camera.position.set( 10, 10, 20 );
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 0xffffff );
+
+				// Ground plane
+
+				const material = new THREE.MeshStandardNodeMaterial();
+
+				const gridXZ = Fn( ( [ gridSize = float( 1.0 ), dotWidth = float( 0.1 ), lineWidth = float( 0.02 ) ] ) => {
+
+					const worldPos = positionWorld;
+					const grid = fract( worldPos.xz.div( gridSize ) );
+					
+					// Distance-based antialiasing
+					const distToCamera = length( worldPos.sub( cameraPosition ) );
+					const smoothing = clamp( distToCamera.div( 100.0 ), 0.01, 0.05 );
```

Comment:
Maybe changing the value from `0.05` to `0.02` will make it a bit sharper, if you prefer.